crates/sui-graphql-rpc/schema/draft_target_schema.graphql

```diff
@@ -1275,8 +1276,6 @@ type ExecutionResult {
 type DryRunResult {
   transaction: TransactionBlock
   error: String
-
-  events: [Event!]
```

Comment:
I removed events since we can get the events from transaction -> effects -> events